code/game/machinery/engi_points_vendor.dm

```diff
@@ -217,23 +218,24 @@
 	exploding = TRUE
 	update_icon()
 	for(var/mob/M in player_list)
-		M << 'sound/machines/Alarm.ogg'
+		send_to_playing_players('sound/machines/Alarm.ogg')
 	var/turf/startpoint = get_turf(src)
 	sleep(100)
 	for(var/mob/M in player_list)
-		M << 'sound/effects/explosionfar.ogg'
+		send_to_playing_players('sound/effects/explosionfar.ogg')
```

Comment:
Again, if we're concerned about efficiency I don't see the point in refactoring that so people standing closer can get a different explosion played. The nuke does the same thing since people don't really care how loud the nuke explosion is based on their relative distance.